mne/source_estimate.py

```diff
@@ -849,11 +849,20 @@ def resample(self, sfreq, npad="auto", window="boxcar", n_jobs=None, verbose=Non
 
         Note that the sample rate of the original data is inferred from tstep.
         """
+        o_sfreq = 1.0 / self.tstep
+        if sfreq == o_sfreq:
```

Comment:
Here I would use `if np.isclose` because the division with `float32` could be inexact